server/container_reopen_log.go

```diff
@@ -40,7 +40,7 @@ func (s *Server) ReopenContainerLog(ctx context.Context, req *pb.ReopenContainer
 	}
 
 	controlPath := filepath.Join(c.BundlePath(), "ctl")
-	controlFile, err := os.OpenFile(controlPath, unix.O_WRONLY, 0)
+	controlFile, err := os.OpenFile(controlPath, syscall.O_WRONLY, 0)
```

Comment:
Can we use [the `os.O_WRONLY` synonym][1] to avoid pulling in the [frozen `syscall` package][2]?

[1]: https://golang.org/pkg/os/#pkg-constants
[2]: https://golang.org/pkg/syscall/#pkg-overview

Reply:
oh yeah. lemme switch to that.